cdap-spark-core/src/test/java/co/cask/cdap/app/runtime/spark/SparkTransactionServiceTest.java

```diff
@@ -173,7 +176,8 @@ public Transaction startLong() {
     };
     txManager.startAndWait();
     try {
-      SparkTransactionService sparkTxService = new SparkTransactionService(new InMemoryTxSystemClient(txManager));
+      SparkTransactionService sparkTxService = new SparkTransactionService(
```

Comment:
It shouldn't be not available in both standalone (set by cdap-default.xml) and in distributed mode (set by YARN, which get passed from the `AbstractProgramTwillRunnable`).